Calculator/Calculator.cpp

```diff
@@ -194,51 +419,120 @@ double expression()
     Token t = ts.get(); 
     while (true) {
         switch (t.kind) {
-        case '+':
-            left += term(); 
-            t = ts.get();
-            break;
-        case '-':
-            left -= term(); 
-            t = ts.get();
-            break;
-        default:
-            ts.putback(t); 
-            return left; 
+            case '+':
+            {
+                left += term();
+                t = ts.get();
+                break;
+            }
+            case '-':
+            {
+                left -= term();
+                t = ts.get();
+                break;
+            }
+            case ',':
```

Comment:
what the hell is this?

Reply:
:))) oops. that was some weird pseudo code i used and forgot to take out